### PR TITLE
Solon snippets

### PR DESCRIPTION
**Pull request by isc-solon**
This PR improves some snippets and adds some new ones. The details are in the two commits.

### CODE REVIEW

File: snippets/objectscript-class.json
Comment (gjsjohnmurray) [inline snippets/objectscript-class.json:19]:
```suggestion
    "prefix": "Property",
```

**Review by gjsjohnmurray — CHANGES_REQUESTED** [review]
Looking good.

In addition to my suggested change to the prefix of the Property snippet, please reinstate the 2-space indents of snippet properties in place of the tabs you seem to be using, e.g.
```
  "Method definition": {
	"prefix": "Method",
	"body": ["Method $1($2) As ${3:%Status}", "{", "\tset $4 = \\$\\$\\$OK", "\t$0", "\treturn $4", "}"]
  },
```
becomes
```
  "Method definition": {
    "prefix": "Method",
    "body": ["Method $1($2) As ${3:%Status}", "{", "\tset $4 = \\$\\$\\$OK", "\t$0", "\treturn $4", "}"]
  },
```

**Comment by isc-solon** [comment]
I think I got everything

**Review by gjsjohnmurray — DISMISSED** [review]
👍

**Comment by gjsjohnmurray** [comment]
@daimor I'll leave it to you to review (or not) and then to do the merge.

**Comment by isc-solon** [comment]
Once this is merged, I have another proposal for _objectscript-class.json_. I'd like to add `/// $1` to the top of **all** of the snippets so that all the class members include documentation. Which means that all the other $#s would have to be "incremented" by 1. Do you like that idea?

**Comment by isc-solon** [comment]
Don't approve yet! I just thought of one more thing...

**Review by gjsjohnmurray — APPROVED** [review]
👍

**Pull request merged**
